### PR TITLE
T-40 - Create hangfire infrastructure

### PR DESCRIPTION
**Pull request by AntonBessarabov**
Implemented hangfire infrastructure

### CODE REVIEW

File: PortfolioKeeper/App_Start/Startup.cs [hunk 1]
```diff
@@ -12,11 +17,24 @@ public partial class Startup
...
             DependencyResolver.SetResolver(new AutofacDependencyResolver(container));
             app.UseAutofacMiddleware(container);
             app.UseAutofacMvc();
 
             ConfigureAuth(app);
+
+            GlobalConfiguration.Configuration.UseAutofacActivator(container, false);
+            GlobalConfiguration.Configuration
+                .UseSqlServerStorage("PortfolioKeeperHangFireDbConnection");
+
```
Comment (stas-litvinenka) [inline PortfolioKeeper/App_Start/Startup.cs]: no need in extra line here

File: PortfolioKeeper/Global.asax.cs [hunk 2]
```diff
@@ -2,13 +2,20 @@
...
 using System.Web.Routing;
+using PortfolioKeeper.Data.Context;
 
 namespace PortfolioKeeper
 {
     public class MvcApplication : HttpApplication
     {
         protected void Application_Start()
         {
+            //todo: fix for crash when database is not exist
```
Comment (stas-litvinenka) [inline PortfolioKeeper/Global.asax.cs]: Better introduce fix here

**Comment by AntonBessarabov** [comment]
[T-40 Create hangfire infrastructure](https://trello.com/c/clmVltMF/40-t-40-create-hangfire-infrastructure)

File: PortfolioKeeper/Global.asax.cs (hunk 2 above)
Comment (AntonBessarabov) [reply PortfolioKeeper/Global.asax.cs]: Sorry, I mean that this block corrects an error (if database is not exist)

**Pull request merged**